packages/tomd/tests/test_fallback.py

```
@@ -0,0 +1,195 @@
+from tomd.api import (
+    _apply_metadata_fallback,
+    _reorder_yaml_body,
+    _remove_yaml_key,
+)
+
+
+class TestReorderYamlBody:
+    def test_already_ordered(self):
+        body = "title: T\ndocument: P0001R0\ndate: 2026-01-01\nrevision: 0"
+        result = _reorder_yaml_body(body)
+        keys = [l.split(":")[0] for l in result.splitlines() if l]
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)